docs/setup.md

```diff
@@ -1,8 +1,44 @@
 # Getting Started
 
+## Know the basics
+
+### Projects
+
+A project represents one application and it has a Git repo. A project can be deployed to multiple environments.
+
+### Environments
+
+An environment is deployment environment such as production, staging, testing. An environment can be associated with many Deploy Groups.
+
+### Deploy Groups
+
+Each environment has many deploy groups. A deploy group is a separate group of machines. For example, a deploy group might represent host machines in a region.
+
+Deploy groups are shared by all projects.
+
+### Stages
+
+A Stage deploys a project to many deploy groups and often maps to 1 environment. While deploying an application to production (for example), one might like to roll it out to different deploy groups in certain order (such as from less sensitive to more sensitive), can also be automated with a [pipeline](/plugins/pipelines/README.md).
+
+### Release
+
+A release represents a git tag like "v1".
```

Comment:
A release isn't really referenced outside of this definition. Could you call it a tagged build?

Reply:
@yifeiwu can we say: `A Release represents a build release with a unique release number similar to git tags, such as "v1".`